api/src/main/java/org/openmrs/api/db/DelegatingFullTextSession.java

```diff
@@ -24,30 +24,32 @@
 import org.springframework.context.ApplicationEventPublisher;
 
 /**
- * Custom implementation of the {@link FullTextSession} interface that acts a wrapper around a
- * target FullTextSession instance, it actually delegates all the method calls directly to the
- * target except for the {@link FullTextSession#createFullTextQuery(Query, Class[])} method where it
- * first notifies registered listeners of the creation event before returning the newly created
- * {@link FullTextQuery} object. The newly created query object and entity type are passed to the
- * listeners wrapped in a {@link FullTextQueryAndEntityClass} object. <br>
+ * Custom implementation of the {@link FullTextSession} interface that acts a
+ * wrapper around a target FullTextSession instance, it actually delegates all
+ * the method calls directly to the target except for the
+ * {@link FullTextSession#createFullTextQuery(Query, Class[])} method where it
+ * first notifies registered listeners of the creation event before returning
+ * the newly created {@link FullTextQuery} object. The newly created query
+ * object and entity type are passed to the listeners wrapped in a
+ * {@link FullTextQueryAndEntityClass} object. <br>
  * <br>
- * An example use case is that a listener can enable/disable filters on the newly created query
- * object.
+ * An example use case is that a listener can enable/disable filters on the
+ * newly created query object.
```

Comment:
please remove your changes to the javadoc

Reply:
is this javadoc formatted automatically?
did you try not including that in your commit?

Reply:
It is automatically formated